turborepo-tests/integration/tests/run/single-package/with-deps-dry-run.t

```diff
@@ -17,7 +17,7 @@ Check
   Tasks to Run
   build
     Task                           = build\s* (re)
-    Hash                           = 4047a6e65d7dafef
+    Hash                           = 81a933c332d3f388
```

Comment:
are all the hashes changing because we're including some kind of "none" value even when there are no root internal deps?

Reply:
Yes, since we're changing the Capnproto structure definition the hashing of it is shifting even when there aren't any internal deps.